vcpkg/commands/edit.md

```diff
@@ -0,0 +1,56 @@
+---
+title: vcpkg edit
+description: Command-line reference for the vcpkg edit command. Opens a port's folder in a new code editor window.
+author: vicroms
+ms.author: viromer
+ms.date: 7/24/2023
+ms.prod: vcpkg
+---
+# vcpkg edit
+
+## Synopsis
+
+```Console
+vcpkg edit <ports> [--options]
+```
```

Comment:
Would it make sense to list the options in the synopsis for completeness?

i.e., `vcpkg edit <ports> [--all] [--buildtrees]`